sunpy/net/dataretriever/sources/goes.py

```diff
@@ -98,7 +93,6 @@ def post_search_hook(self, i, matchdict):
         rowdict['End Time'] = tr.end
         rowdict['End Time'].format = 'iso'
         rowdict["Instrument"] = matchdict["Instrument"][0].upper()
-        rowdict["SatelliteNumber"] = i["SatelliteNumber"]
```

Comment:
This isn't needed now?

Reply:
yes, since these are values that we passed to the scraper before, and instead of parsing them out from the final url here, i included them in the output row/dict at the places directly where we call this helper function, which are [here](https://github.com/exitflynn/sunpy/blob/skyskraper/sunpy/net/dataretriever/sources/goes.py#L307-L311) and [here](https://github.com/exitflynn/sunpy/blob/skyskraper/sunpy/net/dataretriever/sources/goes.py#L138).

Reply:
this was possible to do earlier since the parse pattern was a whole different thing and the base url's could be visited by the `baseurl` which had values like these filled in. We'll now need the pattern to have these filled in too instead of having placeholders which could extract this from the final url's. This was the initial motivation to do it, though now I think that it makes more sense to do it this way since we already have access to these values at these positions.

Reply:
So if we do not need this here, I can resolve this conversation?

Reply:
yes!